robot_calibration/src/models.cpp

```diff
@@ -41,8 +42,12 @@ ChainModel::ChainModel(const std::string& name, KDL::Tree model, std::string roo
     root_(root), tip_(tip), name_(name)
 {
   // Create a KDL::Chain
-  if (!model.getChain(root, tip, chain_))
-    std::cerr << "Failed to get chain" << std::endl;
+  const auto valid_model = model.getChain(root, tip, chain_);
+  if (!valid_model) {
```

Comment:
To match the existing style, please put opening braces on a new line.
Also, instead of creating "valid_model" - why not just keep the "if (!model....)"?

Reply:
Fixed.